src/components/structures/UserSettings.js

```diff
@@ -379,19 +395,42 @@ module.exports = React.createClass({
                             Disable inline URL previews by default
                         </label>
                     </div>
+                    { settingsLabels.map( setting => {
```

Comment:
'cos we need to actually return an array, and previously the code was broken. forEach doesn't return.